### PR TITLE
Fix a crash introduced by 3d5e9ab by adding a nullptr check.

### PR DESCRIPTION
**Pull request by rniwa**

### CODE REVIEW

**Comment by steakhal** [comment]
Hey, do you think a regression test would be valuable?

**Comment by rniwa** [comment]
> Hey, do you think a regression test would be valuable?

Oh, that's a very good point. Added one.

File: clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp [hunk 1]
```diff
@@ -54,7 +54,7 @@ class UncountedCallArgsChecker
       bool shouldVisitImplicitCode() const { return false; }
 
       bool TraverseDecl(Decl *D) {
-        if (isa<ClassTemplateDecl>(D) && isRefType(safeGetName(D)))
+        if (D && isa<ClassTemplateDecl>(D) && isRefType(safeGetName(D)))
           return true;
         return RecursiveASTVisitor<LocalVisitor>::TraverseDecl(D);
       }
```
Comment (mizvekov) [inline clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp:60]: Shouldn't this be overriding `TraverseTemplateInstantiations(ClassTemplateDecl *D)` instead?
    
 `RecursiveASTVisitor<LocalVisitor>::TraverseDecl`  will ensure you won't be visiting null nodes.

File: clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp [hunk 2]
```diff
@@ -54,7 +54,7 @@ class UncountedCallArgsChecker
       bool shouldVisitImplicitCode() const { return false; }
 
       bool TraverseDecl(Decl *D) {
-        if (isa<ClassTemplateDecl>(D) && isRefType(safeGetName(D)))
+        if (D && isa<ClassTemplateDecl>(D) && isRefType(safeGetName(D)))
```
Comment (steakhal) [inline clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp:57]: Have you considered using "isa_and_nonull"?

File: clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp (hunk 1 above)
Comment (rniwa) [reply clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp]: Good point. That would be cleaner.

**Review by mizvekov — APPROVED** [review]

**Pull request merged**